app/assets/javascripts/short-form/ShortFormApplicationService.js.coffee

```diff
@@ -393,8 +393,6 @@ ShortFormApplicationService = (
     if type == 'liveWorkInSf' || type == 'all'
       Service._updatePreference('liveInSf', Service.liveInSfMembers())
       Service._updatePreference('workInSf', Service.workInSfMembers())
-    else if type == 'workInSf'
```

Comment:
Not sure why this was deleted? Is it because it get's called within `if type == 'liveWorkInSf' || type == 'all'`?

Reply:
it got removed because it didn't seem like we're ever calling it for just `workInSf` any more... in fact, looking again, it actually seems like we really just do it for `liveWorkinSf` or `all`. This is one of those things that could probably be refactored a bit... `checkPreferenceEligibility` gets used as a page callback which then calls this function with `refreshPreferences('liveWorkInSf')`. Otherwise various things in the Service just call `refreshPreferences('all')`. 

I think at some point we were like, hmm if you change your address/etc, it might change around several different requirements... no harm in checking out all your preferences and making sure you're still eligible (the blunt force approach).